.pre-commit-config.yaml

```diff
@@ -1,3 +1,5 @@
+default_language_version:
+  python: python3.7
```

Comment:
Could this be set more generically to `python3` instead of `python3.7`, or does it need to be explicit?

Reply:
I'm fairly sure it could be `python3`, but since we are using 3.7 on this project, I thought I'd set it explicitly and maintain consistency. I'd be happy to change it if you have a strong feeling about it.